app/(home)/plans/[branche]/page.tsx

```diff
@@ -29,18 +29,18 @@ export default async function Page(props: PageProps) {
   }
 
   branche.plans = await getPlans(branche);
-  if (!branche.plans.length) {
+  if (branche.plans.length) {
```

Comment:
```suggestion
  if (!branche.plans.length) {
```

Esse tela é para aparecer quando não tem planos